tests/Integration/tests/markdown/link-page-md/input/subpages/index.md

```diff
@@ -0,0 +1,9 @@
+# Markdown with links
+
+This is a Markdown document with some basic formatting. 
+
+* [Subpage 1](subpage1.md)
+* [Page 1](../page1.md)
+* [Page 1](/page1.md)
+* [Page 1](/page1.html)
```

Comment:
I think this should be invalid, linking to an html page does not really make sense to me.

Reply:
Mmh, wouldnt it make sense to like link to another output page? I think I have seen this used before?

Reply:
See for example: https://tomcam.github.io/least-github-pages/markdown-links.html